roles/security/tasks/suricata.yml

```diff
@@ -71,10 +69,33 @@
   - emerging-inappropriate
   - emerging-imap
   - emerging-voip
+  - tor
+  - emerging-p2p
+  - emerging-web_specific_apps
+  - emerging-activex
+  - emerging-web_server
+  register: suri_cfg
+  tags:
+  - security
+  - ids
+
+- name: suricata | Disable specific IDS alerts
+  lineinfile: dest=/etc/suricata/rules/{{ item.rules }}.rules
+              regexp='^(alert .*{{ item.alert }}.*)$'
+              line='#\1'
+              backrefs=yes
+              backup=yes
+  with_items:
+    - { rules: 'emerging-policy', alert: 'YUM User-Agent' }
```

Comment:
We could even do defaults in ``roles/security/vars/`` and then *optionally* override them at the host/group level or as arguments to the role.